addons/resource.language.en_gb/resources/strings.po

```diff
@@ -22173,7 +22173,125 @@ msgctxt "#37057"
 msgid "Data chunk size used on SMB connections"
 msgstr ""
 
-#empty strings from id 37058 to 38010
+#empty strings from id 37058 to 37100
+
+#. Settings / Services "Caching" category label
+#: system/settings/settings.xml
+msgctxt "#37101"
+msgid "Caching"
+msgstr ""
+
+#. Description of category #37101 "Caching"
+#: system/settings/settings.xml
+msgctxt "#37102"
+msgid "This category contains the settings for configure Caching of Internet streams, network files and local files."
```

Comment:
This category contains settings that configure caching of local and network files, and internet streams."